conductor/src/main.rs

```diff
@@ -430,6 +430,29 @@ async fn run(metrics: CustomMetrics) -> Result<(), ConductorError> {
                 let spec_js = serde_json::to_string(&current_spec.spec).unwrap();
                 debug!("dbname: {}, current_spec: {:?}", &namespace, spec_js);
 
+                if read_msg.message.event_type == Event::Stop {
```

Comment:
Didn't move this into its own function because there are too many params that gets passed to the function and it becomes messy. We should at some point refactor the entire function and break it down into multiple smaller functions.